wkhtmltopdf-image/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+################
+# Docker file to build a python container hosting the wkhtmltopdf service.
+################
+
+#Based off of the ubuntu container
+FROM ubuntu:xenial
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get install -y \
+    bash wget build-essential xorg libssl-dev libxrender-dev python-pip
+
+RUN pip install werkzeug executor gunicorn
+
+#Install wkhtmltopdf
+RUN wget -O wkhtmltox.tar.xz https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz
+RUN tar xvfJ wkhtmltox.tar.xz && rm wkhtmltox.tar.xz
+RUN ln -s /wkhtmltox/bin/wkhtmltopdf /usr/local/bin/wkhtmltopdf
```

Comment:
This lets us easily bump the version of wkhtmltopdf as new releases become available.